projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss

```diff
@@ -48,7 +48,7 @@
         flex-grow: 1;
 
         @if $variant == 'indigo' {
-            box-shadow: elevation(3), 0 0 0 rem(1px) color(null, 'gray', if($theme-variant == 'light', 400, 100));
+            box-shadow: elevation(if($theme-variant == 'light', 3, 2)), 0 0 0 rem(1px) color(null, 'gray', if($theme-variant == 'light', 400, 100));
```

Comment:
This should ideally come from the schemas, the same way it's set for the drop-down component schema.

Reply:
I can't do anything unless we want to expose new variables for all themes.